fmdtools/analyze/graph.py

```diff
@@ -1032,96 +1159,182 @@ def data_average(data):
         list_average.append(sum(x[i] for x in data)/len(data))
     return list_average
 
-def data_error(data,average):
-    """Calculates error for each column in data"""
+def data_error(data, average):
+    """
+    Calculates error for each column in data
+
+    Parameters
+    ----------
+    data
```

Comment:
data : list
   List of lists from sff_model
average : list
   Average of data generated from sff_model over time

Returns
----------
lower_error : float
    Lower bound of error 
upper_error : float 
    Upper bound of error